### PR TITLE
Implement Login

### PR DESCRIPTION
**Pull request by ErnestasKaralius**
Added Login screen where you can input the username to use in the app, username display on the nav-bar, messages will be automatically sent with the current username, removed Privacy.cshtml and _LoginPartial.cshtml.

### CODE REVIEW

File: CollabApp/CollabApp.mvc/Views/Post/Index.cshtml [hunk 1]
```diff
@@ -13,10 +13,6 @@
     <h1>Posts</h1>
     <div class="container">
         <form asp-asp-controller="Post" asp-action="Index" method="post">
-            <div class="row">
-                <div class="col-1">Author</div>
```
Comment (niasenago) [inline CollabApp/CollabApp.mvc/Views/Post/Index.cshtml:17]: dabar posto autorius niekur nėra set'inamas.
image

**Review by niasenago — CHANGES_REQUESTED** [review]

File: CollabApp/CollabApp.mvc/Views/Post/Index.cshtml (hunk 1 above)
Comment (ErnestasKaralius) [reply CollabApp/CollabApp.mvc/Views/Post/Index.cshtml:17]: It should be fixed now.

File: CollabApp/CollabApp.mvc/Views/Home/Chat.cshtml [hunk 2]
```diff
@@ -12,10 +12,6 @@
     <h1>Welcome to the chat!</h1>
 
     <div class="container">
-        <div class="row">
-            <div class="col-1">User</div>
```
Comment (niasenago) [inline CollabApp/CollabApp.mvc/Views/Home/Chat.cshtml:16]: now i can send only empty messages
[cinnamon-2023-09-30T164751+0300.webm](https://github.com/niasenago/PSI-project/assets/118674771/8dd7519a-e4aa-43c1-9154-ae74d17cbded)

**Review by niasenago — CHANGES_REQUESTED** [review]

File: CollabApp/CollabApp.mvc/Views/Home/Chat.cshtml (hunk 2 above)
Comment (ErnestasKaralius) [reply CollabApp/CollabApp.mvc/Views/Home/Chat.cshtml:16]: Let's hope this is the last bug

Comment (niasenago) [reply CollabApp/CollabApp.mvc/Views/Home/Chat.cshtml:16]: bug is solved

**Review by niasenago — APPROVED** [review]

**Pull request merged**